src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/Length.kt

```diff
@@ -0,0 +1,7 @@
+package uk.gov.justice.digital.hmpps.hmppsintegrationapi.models
+
+data class Length(
+  val duration: Int? = null,
+  val units: String? = null,
+  val terms: List<Term> = listOf(Term()),
```

Comment:
```suggestion
  val terms: List<Term> = emptyList(),
```
If there are no terms, then I don't think we should be defaulting to a list with a single object with all null properties...